File: beginner/c/2164.c
#include <stdio.h>
#include <math.h>

int main() {
    int n;

    scanf("%d", &n);

    double result = (pow((((1 + sqrt(5)) / 2)),n) - pow((((1 - sqrt(5)) / 2)),n)) / sqrt(5);

    printf("%.1lf\n", result);

    return 0;
}

File: beginner/java/2164.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int n = scanner.nextInt();

        double result = (Math.pow((((1 + Math.sqrt(5)) / 2)),n) - Math.pow((((1 - Math.sqrt(5)) / 2)),n)) / Math.sqrt(5);

        System.out.printf("%.1f%n", result);
        

        scanner.close();
    }
}
